program/src/processor/mod.rs

```diff
@@ -0,0 +1,152 @@
+use borsh::BorshDeserialize;
+use solana_program::{
+    account_info::AccountInfo, entrypoint::ProgramResult, msg, program_error::ProgramError,
+    pubkey::Pubkey,
+};
+
+use crate::instruction::{
+    accounts::{
+        DeactivateStakeAccounts, DistributeRewardsAccounts, HarvestHolderRewardsAccounts,
+        HarvestStakeRewardsAccounts, InactivateStakeAccounts, InitializeConfigAccounts,
+        InitializeStakeAccounts, SetAuthorityAccounts, SlashAccounts, StakeTokensAccounts,
+        UpdateConfigAccounts, WithdrawInactiveStakeAccounts,
+    },
+    Instruction,
+};
```

Comment:
Ohhh yeah we gotta clean these imports up!

Reply:
This will get fixed once I setup the `nightly fmt`, right?

Reply:
I just added the nightly-only configs to my repo's `rustfmt.toml` and have been running `cargo +nightly fmt`.

Reply:
I will switch to nightly when I set up the CI.